cfgov/jinja2/v1/_includes/molecules/contact-email.html

```diff
@@ -15,9 +15,12 @@
 
    ========================================================================== #}
 
+{# TODO: Is this used anywhere? This pattern makes no sense in it's current form inside expandables #}
```

Comment:
@Scotchester Looks like this is used in the info units? https://github.com/cfpb/cfgov-refresh/search?utf8=%E2%9C%93&q=contact-email.html&type=

Reply:
Interesting. Not sure how I missed that. And not sure how we could easily tell if this within-info-unit usage is actually happening...

Reply:
I have no clue other than asking content editors if they've used it. Think it's safe to remove the note?